internal/manifest/tar.go

```diff
@@ -10,24 +10,27 @@ type Tar struct {
 	Base
 	Filename string
 
-	inputPipeline *Base
+	Format   osbuild.TarArchiveFormat
+	RootNode osbuild.TarRootNode
+	ACLs     *bool
+	SELinux  *bool
+	Xattrs   *bool
+
+	inputPipeline Pipeline
 }
 
 // NewTar creates a new TarPipeline. The inputPipeline represents the
 // filesystem tree which will be the contents of the tar file. The pipelinename
 // is the name of the pipeline. The filename is the name of the output tar file.
 func NewTar(m *Manifest,
 	buildPipeline *Build,
-	inputPipeline *Base,
+	inputPipeline Pipeline,
 	pipelinename string) *Tar {
 	p := &Tar{
 		Base:          NewBase(m, pipelinename, buildPipeline),
 		inputPipeline: inputPipeline,
 		Filename:      "image.tar",
 	}
-	if inputPipeline.manifest != m {
-		panic("tree pipeline from different manifest")
-	}
```

Comment:
Should this be replaced by anything? Again, not worth a rebase.

Reply:
Yes it's unfortunate that we lost the sanity checks but we could figure out a new way to do it.